### DISCUSSION TITLE
[5주차 알고리즘 스터디] 정렬

### DISCUSSION THREAD

**Discussion by bigsaigon333** [fe-backpackers/js-algorithm-basecamp#59] [General]
### 회고 순서
체크인 : ~22:05
다른 사람 코드 참고하여 나의 코드 개선 && 자유로운 의견교환: ~22:40
개인 회고: ~22:50
팀별 공유: ~23:00

**Comment by bigsaigon333** [comment]
## 체크인 : ~22:05

지금의 기분을 10점 만점으로 표현해주세요~

**Reply by sooster910** [reply]
7점: 카드정렬하기 문제를 못풀어서 맘에 걸리네요

**Reply by bigsaigon333** [reply]
7점: 모의면접 스터디 피드백 받은 내용이 날라가서 마음이 아픕니다.. 대충 복구하긴 했지만 :sob:  회고 끝나고 누나집에 가야해서 마음이 급하네요 :dizzy_face:

**Reply by Yeonlisa** [reply]
7점 : 세련되게 코드를 작성하고싶네요

**Reply by ganeodolu** [reply]
7점: 힙큐를 그냥 넘겨서 찜찜하네요ㅎㅎㅎ

**Comment by bigsaigon333** [comment]
## 개인 회고 & 체크아웃

**Reply by sooster910** [reply]
8점  :  새로 시작한  문제지정해서 이야기나누는 방법 해보았는데 도움이 되는 것 같습니다.

**Reply by bigsaigon333** [reply]
- 느낀점: 앞으로 스터디 때 남의 코드를 리뷰하는 것보다, 남의 코드를 보고 자기 코드를 개선해보는 것에 좀 더 초점을 맞추는 식으로 진행해보려고 해요. 남의 코드 보면서 궁금한 점 물어봐서 이를 자기 것으로 만들어, 궁극적으로 자기 코드에 반영되어야 진정한 학습이 아닐까 싶어요

- 생각보다 오늘 회고시간이 좀 정적이어서 다들 어떠셨는지 궁금하네요. 아무런 피드백이라도 남겨주시면 감사하겠습니다~

- 체크아웃: 8점! 힙큐 설명하면서, 다시 한번 힙큐 원리에 대해서 떠올리는 계기가 되었습니다. 공유해주신 유튜브 영상도 시청해야겠습니다!

**Reply by Yeonlisa** [reply]
7점 : 새로운 진행방식이 괜찮은것 같습니다

**Reply by ganeodolu** [reply]
8점 : 꾸준히 참가에 의의와 이진탐색도 열심히 해보겠습니다ㅎㅎ2021년도 두달밖에 안남았다니😱
